app/invocation/cache_requests_card.tsx

```diff
@@ -611,6 +636,24 @@ export default class CacheRequestsCardComponent extends React.Component<CacheReq
         className={
           this.getGroupBy() === cache.GetCacheScoreCardRequest.GroupBy.GROUP_BY_TARGET ? "group-by-target" : ""
         }>
+        <Modal
+          className="workflow-upsell-modal"
+          isOpen={this.state.isWorkflowUpsellOpen}
+          onRequestClose={this.onCloseWorkflowUpsell.bind(this)}>
+          <Dialog>
+            <DialogHeader>
+              <DialogTitle>GitHub Sync Required</DialogTitle>
+            </DialogHeader>
+            <DialogBody>
+              {/*TODO(Maggie): Link to a remote bazel blog post*/}
+              <p>To use Remote Bazel backed UI features, you must link a GitHub repository.</p>
+            </DialogBody>
+            <DialogFooter>
+              <OutlinedButton onClick={this.onCloseWorkflowUpsell.bind(this)}>Cancel</OutlinedButton>
+              <Button onClick={this.onClickWorkflowUpsell.bind(this)}>Link a Repository</Button>
```

Comment:
nit: use `LinkButton` here instead of manually calling `window.open()` - it behaves the same as `<a>` so you can set `target` as well.